Tools/DumpRenderTree/mac/DumpRenderTree.mm

```diff
@@ -864,6 +864,7 @@ static void setWebPreferencesForTestOptions(WebPreferences *preferences, const W
                     && ![feature.key isEqualToString:@"NeedsSiteSpecificQuirks"]
                     && ![feature.key isEqualToString:@"BeaconAPIEnabled"]
                     && ![feature.key isEqualToString:@"LocalFileContentSniffingEnabled"]
+                    && ![feature.key isEqualToString:@"HTTPSByDefaultEnabled"]
```

Comment:
Why do we need to enable this in DumpRenderTree?

Reply:
This is actually disabling `HTTPSByDefaultEnabled`. We are doing the same in WebKitTestRunner. Without this change, requests to `web-platform.test` are being upgraded to https, and the load fails.